source/codegen/CodeGenFunction.cpp

```diff
@@ -71,6 +71,8 @@ llvm::Value* CodeGenFunction::emit(const Instr& instr) {
             return emitStore(instr.getOperands()[0], instr.getOperands()[1]);
         case InstrKind::negate:
             return emitNegate(instr.type, instr.getOperands()[0]);
+        default: // Suppress clang -Wswitch
```

Comment:
No need for the comment here, I should have just added this default case. This code isn't being built by CI builds right now.

Reply:
Yes, I remove the comment.